src/constants.js

```diff
@@ -0,0 +1,14 @@
+/* name constants as NOUN_ACTION so that actions related
+   by nouns will show up near each other in an alphabetized
+   list
+*/
+
+const GLOSSARY_HIDE = 'GLOSSARY_HIDE'
+const GLOSSARY_SHOW = 'GLOSSARY_SHOW'
+const GLOSSARY_SHOW_TERM = 'GLOSSARY_SHOW_TERM'
```

Comment:
a pattern i've seen which is perhaps a little terser is to just export each const line individually, i.e.,
```
export const GLOSSARY_HIDE = 'GLOSSARY_HIDE'
export const GLOSSARY_SHOW = 'GLOSSARY_SHOW'
export const GLOSSARY_SHOW_TERM = 'GLOSSARY_SHOW_TERM'
```

still repeats typing "`export`", but at least it doesn't double the double of lines